package-lock.json

```diff
@@ -9107,15 +9094,6 @@
         "node": ">=12"
       }
     },
-    "node_modules/dynamic-dedupe": {
```

Comment:
out of interest why are so many node_modules being removed here? were they used for the previous Error handling fixtures and we do not need them anymore?

Reply:
I think they were used by `ts-node-dev`, which I removed as it was only used in the old `npm run dev` command (depcheck flagged it to me)